src/sentry/models/actor.py

```diff
@@ -295,6 +295,30 @@ def from_actor_identifier(cls, actor_identifier: int | str | None) -> ActorTuple
         except IndexError as e:
             raise serializers.ValidationError(f"Unable to resolve actor identifier: {e}")
 
+    @classmethod
+    def from_id(cls, user_id: int | None, team_id: int | None) -> ActorTuple | None:
+        from sentry.models.team import Team
+        from sentry.models.user import User
+
+        if user_id and team_id:
+            raise ValueError("user_id and team_id may not both be specified")
+        if user_id and not team_id:
+            return cls(user_id, User)
+        if team_id and not user_id:
+            return cls(team_id, Team)
+
+        return None
+
+    @classmethod
+    def from_ids(cls, user_ids: Sequence[int], team_ids: Sequence[int]) -> Sequence[ActorTuple]:
+        from sentry.models.team import Team
+        from sentry.models.user import User
+
+        return [
+            *[cls(user_id, User) for user_id in user_ids],
+            *[cls(team_id, Team) for team_id in team_ids],
+        ]
```

Comment:
Some helpers to make working with the actors list in our serializer a bit easier.

I couldn't find how we were doing this in other places, but this seems to make sense.